lib/src/models/user.dart

```diff
@@ -23,8 +25,9 @@ class User {
       : id = json['id'],
         email = json['email'],
         firstName = json['firstName'] ?? '',
-        languages = (json['language'] ?? [] as List<String>).map((v) => Language.fromString(v)).cast<Language>().toList(),
+        languages = (json['language'] ?? [] as List<String>).map((v) => Language.fromString(v)).cast<Language>().toList(growable: false),
```

Comment:
I believe this was left as "growable: true" to make it easier to add or remove languages, but I added since the `cloneWith` function which makes it a lot easier to work with immutability.

I'll mark this as a breaking change in the new API version.